.github/workflows/build.yml

```diff
@@ -4,39 +4,25 @@ on:
   push:
     branches:
       - master
+    tags:
+      - '*'
   pull_request:
     branches:
       - master
 
 env:
-  DOCKER_HOSTNAME: ghcr.io
   GO_VERSION: 1.13
 
 jobs:
   build:
-    name: Build and push img
+    name: Build
```

Comment:
we have 2 rules with the same name `Build`

Reply:
I can rename the workflow name (and file) to test